### DISCUSSION TITLE
Cant get Bot to keep Items

### DISCUSSION THREAD

**Discussion by Borega** [isid0re/SoloLeveling#121] [Q&A]
Iam trying to get my bot to keep certain Items but he sells them all no matter if I add a pickit or not. I can not get him to keep certain Items. He picks them up but sells them at a vendor every time. Any way to stop this?

**Comment by isid0re** [comment]
Thanks for posting this. How old is the version of sololeveling you are running? 

I've recently done a bunch of changes to logging and clearinventory with the pickit. I'd like to know so I can either track down where the problem lay or ask you to update the sololeveling files.

**Reply by Borega** [reply]
I downloaded it a few days ago so its somewhat recent. Maybe you could enlighten me: I see that he seems to want to keep as little stuff as possible. But why not whitelist the .nip files one adds?

**Reply by Borega** [reply]
managed to get it working by adding the items to wanted items part.

**Reply by theBGuy** [reply]
Could you try this with a fresh download of sololeveling? I've tried replicating your issue but mine keeps items that are listed on the pickit
